Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1457,6 +1457,22 @@ final class StatusTableViewController: LoopChartsTableViewController {
                 self.presentSimulatedCoreDataMenu()
             })
         }
+        if FeatureFlags.mockTherapySettingsEnabled {
+            actionSheet.addAction(UIAlertAction(title: "Mock Therapy Settings", style: .default) { _ in
+                let settings = TherapySettings.mockTherapySettings
+                self.deviceManager.loopManager.settings.glucoseTargetRangeSchedule = settings.glucoseTargetRangeSchedule
```

Comment:
[nit] Not sure if we want to allow doing this more than once.  That said, this is a debug menu item, so user should (hopefully) know what they're doing

Reply:
I could see this as being a useful way to set settings back to a specific state if needed, though - like QA was making changes for 1 test and needs to reset it back to a particular state.